sdk/python/kfp/dsl/_container_op.py

```diff
@@ -1240,6 +1241,9 @@ def _decorated(*args, **kwargs):
 
     self.pvolumes = {}
     self.add_pvolumes(pvolumes)
+    
+    self.cpu_request = None
+    self.memory_request = None
```

Comment:
nit: How about making these readonly properties, so that they can't be modified other than calling `add_cpu_request` and `add_memory_request`?

Reply:
Sure, update it.